github_actions/lib/dependabot/github_actions.rb

```diff
@@ -15,6 +15,7 @@
   register_label_details(
     "github_actions",
     name: "github_actions",
+    description: "Pull requests that update GitHub action code",
     colour: "000000"
   )
```

Comment:
I wanted to test this registration code directly but it looks like we do some work to skip or remove this registration in the tests in favour of https://github.com/dependabot/dependabot-core/blob/44168cffbace107ee1021713fb71eed53ebc6e56/common/spec/dummy_package_manager/dummy.rb#L9. Because of this I had to stub out this registration in the tests added below.